app/assets/stylesheets/footer.scss

```diff
@@ -19,10 +19,12 @@
     position: relative;
     flex-direction: column;
     padding: 0 var(--su-2);
+    margin-bottom: calc(var(--su-9) + env(safe-area-inset-bottom));
```

Comment:
what's `env` ?

Reply:
It's like a system predefined variable (constant?). There's bunch of them, but this particular one is about extra space when you start scrolling a webpage on iOS browser and the address bar and bottom bar disappear making the page "fullscreen". And this env() value defines extra padding for the notch and home indicator: https://css-tricks.com/the-notch-and-css/